src/main/java/com/dougnoel/sentinel/steps/DownloadVerificationSteps.java

```diff
@@ -61,6 +63,34 @@ public static void verifyFileWithFilenameDownloaded(String expectedFilename) thr
 		assertTrue(expectedResult, StringUtils.equals(downloadedFilename, expectedFilename));
 	}
 
+	/**
+	 * Uses the DownloadManager to fetch the most recently downloaded file path.
+	 * Then verifies the filename contains the current date time in <b>YYYY_MMM_dd</b> format by default or a supplied format.
+	 * <p>
+	 * <br><b>Gherkin Example:</b><br>
+	 * <ul>
+	 * <li>I verify the filename contains today's date</li>
+	 * <li>I verify the filename contains today's date <i>formatted as <b>YYYY_MMM_dd_hh:ss:mm</b></i></li>
+	 * </ul>
+	 * <p>
+	 * @param dateTimePattern String an optional datetime pattern
+	 */
+	@Then("^I verify the filename contains today's date(?: formatted as )?(.*?)$")
+	public static void verifyFileWithDateTimeFilename(String dateTimePattern) {
+		if(dateTimePattern.isBlank())
+			dateTimePattern = "YYYY_MMM_dd";
+
+		DateTimeFormatter dateFormatter = DateTimeFormatter.ofPattern(dateTimePattern);
+		String formattedDateTime = dateFormatter.format(LocalDateTime.now());
+
+		String mostRecentDownloadPath = DownloadManager.getMostRecentDownloadPath().toString();
```

Comment:
this path is the fully qualified path on the system. for example, "C:/source/sentinel/downloads/newdownload.pdf". You should grab just the filename from the path for the comparison, since that's what the step says it does.

Reply:
Aye I'll make the change!